docs/how-to/ccv0.sh

```diff
@@ -421,7 +466,7 @@ open_kata_console() {
 
 open_kata_shell() {
     get_ids
-    sudo kata-runtime exec ${sandbox_id}
+    sudo -E env "PATH=$PATH" kata-runtime exec ${sandbox_id}
```

Comment:
Just curious why it needs `env` here.

Reply:
That's a really good question and one that I don't have an answer to, so I assume I made an error typing in that file rather than the doc or something. I'll remove it now. Thanks for the catch!